drm-ffi/drm-sys/src/lib.rs

```diff
@@ -3,15 +3,27 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 
+#[cfg(any(target_os = "android", target_os = "linux"))]
+extern crate linux_raw_sys;
+
+#[cfg(not(any(target_os = "android", target_os = "linux")))]
+extern crate libc;
```

Comment:
Is this still needed if we were to set edition 2018?

Reply:
No, the `extern` crate lines would be unnecessary in edition 2018 or later.

Reply:
I'm not sure what would limit us from setting it to `2018`, given that the MSRV is already at 1.65` and `drm-sys` can't be compiled with much older Rust anyway?

Reply:
Yeah, with the current MSRV I don't think there's any reason not to update to the current 2021 edition.